src/core/models/quarters.ts

```diff
@@ -0,0 +1,51 @@
+import { Schema, model } from "mongoose";
+
+const QuarterSchema = new Schema({
+    
+    currentYear: { 
+        type: Number,
+        //i was thinking of using date for this field so this can be automated rather than us putting it in manually.
+        //type: Date,
+        min: 2024,
+    },
+
+    currentQuarter: {
+        type: Number,
+        min: 1,
+        max: 4,
+        required: true
+    },
+
+    starting_capital: {
+        type: Float64Array,
+        min: 0
+    },
+
+    ending_capital: {
+        type: Float64Array,
+        min: 0
+    },
+
+    roi: {
+        type: Float64Array,
+        required: true,
+        min: 0
+    },
```

Comment:
Should be a number between 0 and 1. It can be negative if we have a losing quarter

Reply:
word word